src/widget/form/chatform.cpp

```diff
@@ -879,6 +889,52 @@ void ChatForm::loadHistory(QDateTime since, bool processUndelivered)
     chatWidget->verticalScrollBar()->setValue(savedSliderPos);
 }
 
+void ChatForm::onScreenshotCreate()
+{
+    QRect region;
+    int result = 0;
+
+    {
+        ScreenshotDialog screenshot(region);
+        result = screenshot.exec();
+    }
+
+    if (result == QDialog::Accepted)
+    {
+        QScreen *screen = QGuiApplication::primaryScreen();
+        if (screen)
+        {
+            // Have to wait for the rubber band to disappear.
+            // Using flush, sendPostedEvents and processEvents does not work.
+            QTimer::singleShot(100, this, [=]()
```

Comment:
I can't say I like the idea of timeouts but, well, [even KSnapShot seems to do it](https://projects.kde.org/projects/kde/kdegraphics/ksnapshot/repository/revisions/master/entry/regiongrabber.cpp#L44), assuming I'm not misunderstanding their code. Still, worth some research to figure out if there isn't a better way, because there really should be.